src/api/jokesApi.ts

```diff
@@ -1,28 +1,43 @@
 import { AxiosError, AxiosResponse } from "axios";
 import { ChuckJoke } from "../types/ChuckJoke";
 import { jokesClient } from "./httpClient";
-import { PATHS } from "../constans/ApiPaths";
+import { PATHS } from "../constants/ApiPaths";
 
 export const jokesApi = {
-  fetchRandomJokes: async (category?: string, name?: string): Promise<AxiosResponse<ChuckJoke>|AxiosError> => { 
-    try { 
-      const response = await jokesClient.get<ChuckJoke>(PATHS.RANDOM_CHUCK_JOKE_PATH, {
-        params: {
-          ...(category && {category: category}),
-          ...(name && {name: name})
-        } }
-        );
-      return response
+  fetchRandomJokes: async (
+    category?: string,
+    name?: string
+  ): Promise<AxiosResponse<ChuckJoke> | AxiosError> => {
```

Comment:
since you got rid of try/catch here you should delete `|AxiosError` and return only Promise<AxiosResponse<ChuckJoke>. You should parse error as AxiosError directly in try catch